### DISCUSSION TITLE
demo version (not working)

### DISCUSSION THREAD

**Discussion by dopevelli** [buresdv/Cork#107] [General]
I downloaded the demo version from https://corkmac.app and it just seems to load indefinitely upon opening. Is there something I'm doing wrong here?

**Comment by buresdv** [comment]
Does it also happen when you download the demo from [here](https://github.com/buresdv/Cork/releases/download/Demo/Cork.zip)?

**Reply by dopevelli** [reply]
It works, thank you!

**Reply by buresdv** [reply]
No problem, I fixed this bug in the first demo release and I guess forgot to replace the link on the website. Thank you for telling me about it!
